lib/web_ui/dev/test_runner.dart

```diff
@@ -279,8 +308,42 @@ class TestCommand extends Command<bool> with ArgUtils {
   bool get isFirefox => browser == 'firefox';
 
   /// Whether [browser] is set to "safari".
-  bool get isSafariOnMacOS => browser == 'safari'
-      && io.Platform.isMacOS;
+  bool get isSafariOnMacOS => browser == 'safari' && io.Platform.isMacOS;
+
+  /// Due to efficiancy constraints, Chrome integration tests only run on
```

Comment:
I'm not sure the constraint is efficiency as much as lack of resources.

Reply:
done